kimhaejung/.env.sample

```
@@ -0,0 +1,9 @@
+DB_CONNECTION = 
+DB_HOST = 
+DB_USERNAME = 
+DB_PASSWORD = 
+DB_DATABASE = 
+DB_PORT = 
+DB_LOGGING = 
+
+DATABASE_URL=
```

sample 파일에는 각 환경변수에 어떤 값 혹은 타입이 설정되는지 힌트를 줄 수 있도록 아래와 같이 수정해주면 좋을 것 같습니다.

추가로, `app.js` 파일을 보면 express로 서버를 실행하기 위해 port 번호를 `3000`으로 직접 기입해주셨는데
환경변수로 port를 관리하면 다음과 같은 이점이 있습니다.

- port를 변경하고 싶을 때 코드를 직접 수정하는 것이 아닌, 환경변수의 값만 수정하여 유연하게 관리할 수 있습니다.
- 2개 이상의 포트를 사용하는 경우 환경변수로 지정하여 사용 포트 충돌을 방지할 수 있습니다.
- 외부에 직접 포트 번호를 노출하지 않아 보안상 안전합니다.


```suggestion
DB_CONNECTION = mysql
DB_HOST = HOST
DB_USERNAME = USERNAME 
DB_PASSWORD = PASSWORD
DB_DATABASE = DATABASE
DB_PORT = DB_PORT
DB_LOGGING = BOOLEAN

DATABASE_URL= "mysql://username:password@host:port/dbName"

PORT = PORT
```